src/test/regress/expected/multi_partitioning.out

```diff
@@ -74,6 +74,12 @@ SELECT * FROM partitioning_hash_test ORDER BY 1;
   4 |     4
 (4 rows)
 
+-- should not return results when only querying parent
+SELECT * FROM ONLY partitioning_test ORDER BY 1;
```

Comment:
I didn't know that UPDATE/DELETE can also have ONLY. Might be nice to have a simple test for it.

Also, one repartitioning test might be good to have.

(I verified all works fine, this is mostly for future)